src/main/java/sirius/db/mixing/properties/BaseEntityRefProperty.java

```diff
@@ -209,6 +225,14 @@ public void link() {
         if (deleteHandler == BaseEntityRef.OnDelete.CASCADE) {
             getReferencedDescriptor().addCascadeDeleteHandler(this::onDeleteCascade);
         } else if (deleteHandler == BaseEntityRef.OnDelete.SET_NULL) {
+            if (!isNullable()) {
+                Mixing.LOG.WARN("Error in property % of %s. The field is not marked as NullAllowed,"
+                                + " therefore SET_NULL is not a valid delete handler!", this, getDescriptor());
+            }
+            if (entityRef.hasWriteOnceSemantics()) {
+                Mixing.LOG.WARN("Error in property % of %s. The field has write once semantics,"
```

Comment:
```suggestion
                Mixing.LOG.WARN("Error in property %s of %s. The field has write once semantics,"
```
shouldnt this also be a `%s`?